app/components/subcases/side-nav.js

```diff
@@ -0,0 +1,34 @@
+import Component from '@glimmer/component';
+
+export default class SubcasesSideNavComponent extends Component {
+  get items() {
+    let items = [];
+
```

Comment:
this get is called many times, triggering a visual change because decision activity results show "no result yet" everytime you move between subcases.
Ideally, we only have to load this once and rerender when adding/removing subcases or reloading the page